util/travis/script.sh

```diff
@@ -25,7 +25,12 @@ if [[ $PLATFORM == "Unix" ]]; then
 		$CMAKE_FLAGS ..
 	make -j2
 	echo "Running unit tests."
-	../bin/minetest --run-unittests && exit 0
+	if [[ -v VALGRIND ]] && [[ $VALGRIND -eq 1 ]]; then
+		valgrind --leak-check=full --undef-value-errors=no --error-exitcode=9 ../bin/minetest --run-unittests && exit 0
+	else
+		../bin/minetest --run-unittests && exit 0
+	fi
```

Comment:
will fix this, i'm trying to use the trusty image on this CI part to have a more up-to-date valgrind because we have one wrong memleak and valgrind check was fixed in 3.9